README.md

```diff
@@ -1,5 +1,8 @@
 <p align="center">
-<img src="http://docs.frrouting.org/en/latest/_static/frr-icon.svg" alt="Icon" width="20%"/>
+  <img src="http://docs.frrouting.org/en/latest/_static/frr-icon.svg" alt="Icon" width="20%"/></br></br>
+  <a href="https://opensource.fb.com/support-ukraine">
```

Comment:
Well, is there something that is not facebook that you can link to? Something commercially neutral (no such thing as politically neutral on this :D)

Reply:
Ye, good point :smile: Fixed.